react/modules/checkoutHook.ts

```diff
@@ -3,13 +3,13 @@ import { useRuntime } from 'vtex.render-runtime'
 
 export default function useCheckout() {
   const { url: checkoutUrl, major } = useCheckoutURL()
-  const { navigate, rootPath = '' } = useRuntime()
+  const { navigate } = useRuntime()
 
   const goToCheckout = (url: string) => {
     if (major > 0 && url === checkoutUrl) {
       navigate({ to: url })
     } else {
-      window.location.href = `${rootPath}${url}`
+      navigate({ to: url, fallbackToWindowLocation: true })
```

Comment:
Missing rootPath

Reply:
Doesn't `navigate` already add the `rootPath`?